vnext/Microsoft.ReactNative/Modules/AccessibilityInfoModule.cpp

```diff
@@ -35,8 +37,25 @@ void AccessibilityInfo::isTouchExplorationEnabled(
   onSuccess(UiaClientsAreListening());
 }
 
-void AccessibilityInfo::setAccessibilityFocus(double /*reactTag*/) noexcept {
-  // no-op - This appears to be unused in RN
+void AccessibilityInfo::setAccessibilityFocus(double reactTag) noexcept {
+  m_context.UIDispatcher().Post([context = m_context, reactTag = std::move(reactTag)] {
+    auto service = winrt::Microsoft::ReactNative::XamlUIService::FromContext(context.Handle());
+    auto element = service.ElementFromReactTag(int64_t(reactTag));
+
+    if (!element) {
+      return;
+    }
+
+    auto frameworkElement = element.try_as<xaml::FrameworkElement>();
+    xaml::Input::FocusManager::TryFocusAsync(frameworkElement, xaml::FocusState::Programmatic);
+    auto peer = xaml::Automation::Peers::FrameworkElementAutomationPeer::FromElement(frameworkElement);
+
+    if (!peer) {
+      return;
+    }
+
+    peer.RaiseAutomationEvent(xaml::Automation::Peers::AutomationEvents::AutomationFocusChanged);
```

Comment:
won't TryFocusAsync raise this if it was able to?

Reply:
got it! I was originally given the RaiseAutomationEvent code as what would be needed to move narrator focus to the new element, but that on its own failed since the new element actually needs to have focus for narrator to not discard the AutomationEvent as bogus...hence call to tryAsyncFocus was added to move the focus. Make sense that tryAsyncFocus would also take care of alerting narrator of the change.